.github/workflows/auto_label.yml

```
@@ -7,6 +7,7 @@ on:
 jobs:
   auto-label:
     runs-on: ubuntu-latest
+    if: false
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)